CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -82,7 +82,7 @@ class HomeTableViewCell: UITableViewCell {
         
         enrollButton.snp.makeConstraints { make in
             make.top.equalTo(sectionLabel.snp.bottom).offset(12)
-            make.leading.equalTo(container.snp.centerX).offset(9)
+            make.leading.equalTo(containerView.snp.centerX).offset(9)
             make.height.equalTo(24)
             make.trailing.equalToSuperview().inset(16)
             make.bottom.equalToSuperview().inset(16)
```

Comment:
i know this isn't from your PR but can you combine these to `make.trailing.bottom.equalToSuperview().inset(16)`